### ISSUE TITLE
Errors in ASHRAE9012016 test files

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#7290]
Issue overview
--------------
Three of the ASHRAE9012016* test files which are included in the distribution are not being tested on CI:

```
#ADD_SIMULATION_TEST(IDF_FILE ASHRAE9012016_ApartmentMidRise_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
#ADD_SIMULATION_TEST(IDF_FILE ASHRAE9012016_Hospital_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
#ADD_SIMULATION_TEST(IDF_FILE ASHRAE9012016_OutPatientHealthCare_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
```

ASHRAE9012016_OutPatientHealthCare_Denver and ASHRAE9012016_ApartmentMidRise_Denver do not run in v9.1.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus 9.1

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by Nigusse** [comment]
@MJW I run the three test files. Test file ASHRAE9012016_Hospital_Denver.idf did not fail but the other two failed. I will correct the two files only.

**Comment by Myoldmopar** [comment]
Fixed via #7351
